tests/e2e/suite_test.go

```diff
@@ -125,8 +125,11 @@ func saveOperatorLogs(buf bytes.Buffer, specName string, output io.Writer) {
 			fmt.Fprintln(output, "ERROR while closing file:", err)
 		}
 	}()
-	for scanner.Scan() {
+
+	lineIdx := 0
+	for scanner.Scan() && lineIdx < capLines {
```

Comment:
We should display the last lines, not the initial part

Reply:
Fixed ...